back-end-102/final-project/src/main.py

```diff
@@ -1,18 +1,290 @@
-from typing import Union
-
-from fastapi import FastAPI
+import bcrypt
+import jwt
+from uuid import uuid4
+from typing import Union, Optional, List
+from pydantic import BaseModel, EmailStr
+from datetime import datetime, timedelta
+from fastapi import FastAPI, Depends, HTTPException
+from fastapi.security import OAuth2PasswordBearer
 
 app = FastAPI()
 
+users = [
+    {"id": 1, "name": "Adilet", "email": "adilet@example.com", "password": "12341234", "role": "admin"},
+    {"id": 2, "name": "Anuar", "email": "anuar@example.com", "password": "12341234", "role": "user"}
+]
+
+projects = []
+
+tasks = []
+
+active_refresh_tokens = {}
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/auth/login")
+
+SECRET_KEY = "BaVArIanMotORWorKs3PoInT5LIterTwINTUrBo"
```

Comment:
ideally should get the secret key from `.env` file, so it is not exposed in the repo

Reply:
.env should be gitignored